s3-proxy/s3-proxy/src/main/java/com/woowacourse/s3proxy/config/WebMvcConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package com.woowacourse.s3proxy.config;
+
+import com.woowacourse.s3proxy.common.fileValidator.FileValidator;
+import com.woowacourse.s3proxy.common.fileValidator.ImageFileValidator;
+import com.woowacourse.s3proxy.config.resolver.ExtensionValidArgumentResolver;
+import java.util.List;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfiguration implements WebMvcConfigurer {
+
+    @Override
+    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
+        resolvers.add(new ExtensionValidArgumentResolver(createFileValidators()));
+    }
+
+    public List<FileValidator> createFileValidators() {
+        return List.of(
+            new ImageFileValidator()
+        );
+    }
```

Comment:
```suggestion
@Configuration
public class WebMvcConfiguration implements WebMvcConfigurer {

    @Override
    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
        resolvers.add(extensionValidArgumentResolver());
    }

    @Bean
    public HandlerMethodArgumentResolver extensionValidArgumentResolver() {
        return new ExtensionValidArgumentResolver(createFileValidators());
    }

    @Bean
    public List<FileValidator> createFileValidators() {
        return List.of(
            new ImageFileValidator()
        );
    }
}
```

* FileValidatorConfiguration에도 ``List<FileValidator>`` Bean이 등록되어있는데요.
* ArgumentResolver를 Bean으로 등록하면 WebMvcConfig 클래스에 모두 정의할 수 있을 것 같습니다.
  * Web관련 Config 성격을 명확하게 다른 Config와 확실하게 분리하는 것이라면 지금 그대로 가도 좋습니다!